### DISCUSSION TITLE
@types/chrome support for chrome.storage.session

### DISCUSSION THREAD

**Discussion by vietnamese-chowder** [DefinitelyTyped/DefinitelyTyped#60097] [Issues with a @types package]
I am using [@types/chrome] version 0.0.183, and trying to use chrome.storage.session (https://developer.chrome.com/docs/extensions/reference/storage/#property-session) gives me the error `TS2339: Property 'session' does not exist on type 'typeof storage'`.

Is there planned support for this property? It already works for me if I edit the generated javascript to replace `chrome.storage.local` with `chrome.storage.session`.

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment]
The `session` storage area seems to be unstable, indicated by the "Pending" badge in the docs.

I'm not sure if the Chrome type definitions are meant to include unstable features, but you can add the following to your TypeScript code in the meantime:

```typescript
declare namespace chrome.storage {
    export var session: StorageArea;
}
```

**Comment by RC-14** [comment]
`chrome.storage.session` types are present in the current version of [@types/chrome].
This should be marked as answered.
